### ISSUE TITLE
inaccurate title bar buttons sizing in cinnamon

### ISSUE THREAD

**Issue by bixty** [vinceliuice/Mojave-gtk-theme#216]
Just installed this excellent gtk theme.
tried for the compact version which works really well as far as some programs concern, like Chromium and Gnome's Epiphany.

i've chosen `Mojave-dark-small` for both Window Borders and Controls in Cinnamon Themes settings but when it comes to something like KDE console, the title bar buttons are the `standard` sized and aren't `compact` like the other programs.

also another problem is with Gnome's Epiphany, where the tab height is not standard when i activate the theme, but when i choose something like Adwaita Dark (for controls) it looks very better and feels more clicky with more height.

Bottom line, i want same size for controls and window borders title buttons and a better look / more height to Gnome Epiphany tabs.

Cinnamon Version: `5.0.6`
